Tests/store_packs.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+import sys
+import argparse
+
+from os import path
+from zipfile import ZipFile, ZIP_DEFLATED
+from Tests.test_utils import run_command, print_error
+from google.cloud import storage
+import google.auth
+
+
+def option_handler():
+    parser = argparse.ArgumentParser(description="Store packs in cloud storage.")
+    parser.add_argument('-a', '--artifactsPath', help="The full path of packs artifacts", required=True)
+    parser.add_argument('-e', '--extractPath', help="Full path of folder to extract wanted packs", required=True)
+    parser.add_argument('-c', '--circleCi', help="Whether run script locally or in circleCi", default=False)
+    parser.add_argument('-p', '--packName', help="Use only in local mode, the target pack name to store.",
+                        required=False, default="")
+    parser.add_argument('-b', '--bucketName', help="Storage bucket name", required=True)
+
+    return parser.parse_args()
+
+
+class PackStorage:
+    CHANGELOG = "changelog.json"
+    PACK_INITIAL_VERSION = "1.0.0"
+
+    def __init__(self, pack_name, pack_path, storage_bucket):
+        self.pack_name = pack_name
+        self.pack_path = pack_path
+        self.storage_bucket = storage_bucket
+
+    @property
+    def latest_version(self):
+        return self._get_latest_version()
+
+    def _get_latest_version(self):
+        if self.CHANGELOG not in os.listdir(self.pack_path):
+            return self.PACK_INITIAL_VERSION
+
+        changelog_path = path.join(self.pack_path, self.CHANGELOG)
+
+        with open(changelog_path, "r") as changelog:
+            changelog_json = json.load(changelog)
+
+            pack_versions = list(changelog_json.keys())
+            pack_versions.sort(key=lambda str_version: [int(v) for v in str_version.split(".")],
+                               reverse=True)
```

Comment:
maybe we should import a package for this, to avoid any errors that may occur?

Reply:
We can use this package https://pypi.org/project/semantic-version/. But I think that we should use it in validation (of semantic versioning) of `changelog.json` instead, which will include all packs versions.

Reply:
👍